htdocs/core/lib/accounting.lib.php

```diff
@@ -347,7 +347,7 @@ function getCurrentPeriodOfFiscalYear($db, $conf, $from_time = null)
 	if ($from_time === null) {
 		$from_time = $now;
 	}
-	$from_db_time = $db->idate($from_time);
+	$from_db_time = dol_print_date($from_time, "%Y-%m-%d");
 
```

Comment:
I agree with @rycks,
$db->idate() is used for SQL request.
Maybe there is a mistake in "$db->escape()" in this SQL request.
Usually db->escape() is not used for dates filters in SQL requests. (only use db->idate()).